### PR TITLE
[411549]  Release Notes Xtend: Multi-catch and multi-type guard included

### PR DESCRIPTION
**Pull request by HBuender**
A release note for multi-catch and multi-type guard support for Xtend

### CODE REVIEW

**Comment by cdietrich** [comment]
Could you add something to the documentation itself too

**Comment by kthoms** [comment]
Good idea. `xtend-website/documentation/202_xtend_classes_members.md` would be the right place.

**Comment by HBuender** [comment]
@kthoms thanks for the hint. I am on it :)

**Comment by HBuender** [comment]
After taking a closer look I would rather put the enhancements in xtend-website/documentation/203_xtend_expressions.md since switch and catch are explained there as well.

File: xtend-website/documentation/203_xtend_expressions.md [hunk 1]
```diff
@@ -919,6 +932,17 @@ val name = try {
   }
 ```
 
+Handling multiple exceptions in a single catch block can be realized using a *multi-catch clause*. The *multi-catch clause* can handle an arbitrary number of exceptions separated by `|`. Within the catch block, the intersection exception type of the exceptions caught by the *multi-catch clause* is available for further processing.
+
+```xtend
+def readFile(String path){
```
Comment (kthoms) [inline xtend-website/documentation/203_xtend_expressions.md]: please use whitespaces around the brackets

File: xtend-website/documentation/203_xtend_expressions.md [hunk 2]
```diff
@@ -919,6 +932,17 @@ val name = try {
   }
 ```
 
+Handling multiple exceptions in a single catch block can be realized using a *multi-catch clause*. The *multi-catch clause* can handle an arbitrary number of exceptions separated by `|`. Within the catch block, the intersection exception type of the exceptions caught by the *multi-catch clause* is available for further processing.
```
Comment (kthoms) [inline xtend-website/documentation/203_xtend_expressions.md]: Please add a note that error is raised when one exception covers another, e.g. `NullPointerException | RuntimeException`

**Comment by kthoms** [comment]
Thx @HBuender

**Pull request merged**